[CIS-1444] Support passing the extra data along with the message (#1722)
GetStream/stream-chat-swift: CHANGELOG.md

# Upcoming

### ✅ Added

> - `ChannelListQuery.membersLimit` param for controlling the number of members returned for each channel [#1721](https://github.com/GetStream/stream-chat-swift/issues/1721).
> - Adds support to pass extra data for message from `ComposerVC` [#1722](https://github.com/GetStream/stream-chat-swift/pull/1722).

Reviewer: We usually don't add the "." at the end here 🤔 
```suggestion
- `ChannelListQuery.membersLimit` param for controlling the number of members returned for each channel [#1721](https://github.com/GetStream/stream-chat-swift/issues/1721)
- Adds support to pass extra data for message from `ComposerVC` [#1722](https://github.com/GetStream/stream-chat-swift/pull/1722)
```

### 🐞 Fixed

> - Fix multiple pagination requests being fired from `ChatChannelVC` and `ChatChannelListVC` [#1706](https://github.com/GetStream/stream-chat-swift/issues/1706).
> - Fix rendering unavailable reactions on `ChatMessageReactionAuthorsVC` [#1719](https://github.com/GetStream/stream-chat-swift/issues/1719).

Reviewer: Same here
```suggestion
- Fix multiple pagination requests being fired from `ChatChannelVC` and `ChatChannelListVC` [#1706](https://github.com/GetStream/stream-chat-swift/issues/1706)
- Fix rendering unavailable reactions on `ChatMessageReactionAuthorsVC` [#1719](https://github.com/GetStream/stream-chat-swift/issues/1719)
```

### ✅ Added
- `ChannelListQuery.membersLimit` param for controlling the number of members returned for each channel [#1721](https://github.com/GetStream/stream-chat-swift/issues/1721)

> - Extra data for message can now be passed directly from `ComposerVC`. In order to pass the extra data for all messages, override the `publishMessage(sender:)` function and adjust the `content.extraData` prior calling `super.publishMessage(sender:)` [#1722](https://github.com/GetStream/stream-chat-swift/pull/1722)

Reviewer 2: nit: let's keep it short? 🙂
```suggestion
- Make it possible to change extra data when editing the message [#1722](https://github.com/GetStream/stream-chat-swift/pull/1722)
```

Author: The `when editing the message` part is misleading. But Ill cut it short :)
